realm-annotations-processor/src/main/java/io/realm/processor/ClassMetaData.java

```diff
@@ -386,6 +410,10 @@ public String getPrimaryKeyGetter() {
         return getters.get(primaryKey.getSimpleName().toString());
     }
 
+    public boolean isNullable(String fieldName) {
```

Comment:
maybe rename to `isFieldNullable` and consider if it should take a VariableElement instead